subprojects/language-native/src/test/groovy/org/gradle/language/plugins/NativeBasePluginTest.groovy

```diff
@@ -411,22 +411,22 @@ class NativeBasePluginTest extends Specification {
     }
 
     def "adds Maven publications for component with main publication"() {
-        def usage1 = Stub(UsageContext)
+        def componentVariant1 = Stub(UsageContext)
         def artifact1 = Stub(PublishArtifact)
         artifact1.getFile() >> projectDir.file("artifact1")
-        usage1.artifacts >> [artifact1]
+        componentVariant1.artifacts >> [artifact1]
         def variant1 = Stub(PublishableVariant)
```

Comment:
🤔  I think this might be clearer and more consistent by calling this `publishableVariant` and using just `variant` for the others.